MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```diff
@@ -98,11 +93,6 @@ private void Start(WebSocketMessageInfo message)
             var dueTimeMs = long.Parse(vals[0], UsCulture);
             var periodMs = long.Parse(vals[1], UsCulture);
 
-            if (vals.Length > 2)
-            {
-                ParseMessageParams(vals.Skip(2).ToArray());
```

Comment:
are you sure no subclasses override this function?

Reply:
Yep, just double checked.